src/components/avatar/index.js

```diff
@@ -46,13 +46,13 @@ const Avatar = (props) => {
           name={ props.name === '' ? 'N/A' : props.name }
         />
       ) }
-      { props.caption && (
+      { props.title && (
```

Comment:
it's not backward compatible but a breaking change. if we release this, we have to change every user avatar in the content editor where we set the name property and set the title property as well.

Reply:
This is what I wanted to avoid if possible

Reply:
@kristof0425 ??